dashboard/app/views/plc/enrollment_evaluations/perform_evaluation.html.haml

```diff
@@ -0,0 +1,18 @@
+=javascript_include_tag 'plc/perform_evaluation'
+
+%h1 Self Evaluation for #{@course.name}
+
+%p
+  This will determine what parts of the course are best suited for your learning needs. There are no wrong answers, just answer as best as you can!
+
+= form_tag(action: 'submit_evaluation') do
+  - @questions.each do |question|
+    %p #{question.question}
+    - answers = question.plc_evaluation_answers
+    - answers.each do |answer|
+      %p
+        %input.radio{type: "radio", value: '' + answer.plc_task_id.to_s, checked: answer == answers.first}
```

Comment:
or alternatively, if you need the to_s, you should be able to take away the addition of empty string because adding an empty string to a string should be a no-op.